lib/adapters/websql/websql.js

```diff
@@ -60,28 +61,6 @@ function fetchAttachmentsIfNecessary(doc, opts, api, txn, cb) {
   });
 }
 
-var cachedDatabases = {};
-
-var sqlitePluginOpenDBFunction =
-  ((typeof navigator !== 'undefined' &&
-    navigator.sqlitePlugin &&
-    navigator.sqlitePlugin.openDatabase) &&
-    navigator.sqlitePlugin.openDatabase.bind(navigator.sqlitePlugin)) ||
```

Comment:
This `navigator.sqlitePlugin` stuff seems to date from an older version of the SQLite Plugin. I'm happy to just require that users use the latest vesion of the SQLite Plugin.